File: src/sdk-reference/php/3/memory-storage/hincrbyfloat/snippets/hincrbyfloat-1.js

<?php

use \Kuzzle\Kuzzle;


$kuzzle = new Kuzzle('localhost');

try {
  $value = $kuzzle->memoryStorage()->hincrbyfloat('key', 'field', 3.14159);
}
catch (ErrorException $e) {

}

File: src/sdk-reference/android/3/memory-storage/hincrbyfloat/snippets/hincrbyfloat-1.java

kuzzle.memoryStorage.hincrbyfloat("key", "field", 3.14159, new ResponseListener<Double>() {
  @Override
  public void onSuccess(int value) {
    // callback called once the action has completed
  }

  @Override
  public void onError(JSONObject error) {
  }
});
